instrumentation/logback/logback-appender-1.0/library/src/test/java/io/opentelemetry/instrumentation/logback/appender/v1_0/OpenTelemetryAppenderConfigTest.java

```diff
@@ -115,13 +114,27 @@ void logWithExtras() {
         .isLessThan(TimeUnit.MILLISECONDS.toNanos(Instant.now().toEpochMilli()));
     assertThat(logData.getSeverity()).isEqualTo(Severity.INFO);
     assertThat(logData.getSeverityText()).isEqualTo("INFO");
-    assertThat(logData.getAttributes().size()).isEqualTo(3);
+    assertThat(logData.getAttributes().size()).isEqualTo(3 + 4); // 4 code attributes
```

Comment:
WDYT about refactoring this to use `OpenTelemetryAssertions` and `AttributesAssert`? Would be way more terse than the current code

Reply:
I was not aware about these classes. Thanks!  I have a first look at the `OpenTelemetryAssertions` and `AttributesAssert` classes. From what I have seen, I have not the feeling that the assertions would be shorter and easier to read  (for example the `assertAttributes` method  of `AttributesAssert` requires a Map). Maybe I could look at this thing more in depth and propose potential improvements in another PR?

Reply:
Sure, that sounds fine 👍